src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -1,10 +1,11 @@
 package com.meilisearch.sdk;
 
 import lombok.Getter;
-import org.json.JSONObject;
+import lombok.Setter;
 
 /** Search request query string builder */
 @Getter
+@Setter
```

Comment:
Are you kidding that `@Setter` config can make a whole builder just by adding the config?

That's why you could remove the `setOffset`, `setQ` and others?

Reply:
By the way, what do you think to have a `setQuery` instead of `setQ`?

Reply:
> Are you kidding that @Setter config can make a whole builder just by adding the config?

This is cool, right? Actually the `setQuery()` was kind of specific but we don't have the usage of it and Yes I think we should rename `setQ` to `setQuery`